libcst/codemod/commands/tests/test_rename.py

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# pyre-strict
+
+from libcst.codemod import CodemodTest
+from libcst.codemod.commands.rename import RenameCommand
+
+
+class TestRenameCommand(CodemodTest):
+
+    ONCALL_SHORTNAME = "instagram_server_framework"
+
+    TRANSFORM = RenameCommand
+
+    def test_rename_name(self) -> None:
+
+        before = """
+            from foo import bar
+
+            def test() -> None:
+                bar(5)
+        """
+        after = """
+            from baz import qux
+
+            def test() -> None:
+                qux(5)
+        """
+
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="foo",
+            orig_object="bar",
+            new_module="baz",
+            new_object="qux",
+        )
+
+    def test_rename_name_asname(self) -> None:
+
+        before = """
+            from foo import bar as bla
+
+            def test() -> None:
+                bla(5)
+        """
+        after = """
+            from baz import qux as bla
+
+            def test() -> None:
+                bla(5)
+        """
+
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="foo",
+            orig_object="bar",
+            new_module="baz",
+            new_object="qux",
+        )
+
+    def test_rename_attr(self) -> None:
+
+        before = """
+            import a.b
+
+            def test() -> None:
+                a.b.c(5)
+        """
+        after = """
+            import d.e
+
+            def test() -> None:
+                d.e.f(5)
+        """
+
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="a.b",
+            orig_object="c",
+            new_module="d.e",
+            new_object="f",
+        )
+
+    def test_rename_attr_asname(self) -> None:
+
+        before = """
+            import foo as bar
+
+            def test() -> None:
+                bar.qux(5)
+        """
+        after = """
+            import baz as bar
+
+            def test() -> None:
+                bar.quux(5)
+        """
+
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="foo",
+            orig_object="qux",
+            new_module="baz",
+            new_object="quux",
+        )
+
+    def test_rename_module(self) -> None:
```

Comment:
This tests a class rename not a module rename. Using an uppercase class name `B` instead of `b` would make this more clear.

Separately, we should test a module rename as well!